test/integration/test_bin.c

```diff
@@ -8,6 +8,8 @@
 
 bool test_rz_bin(void) {
 	RzBin *bin = rz_bin_new();
+	const char* default_algos[] = {"md5", "sha1", "sha256", "crc32", "entropy"};
+	bin->default_hashes = rz_list_new_from_array(default_algos, sizeof(default_algos)/sizeof(default_algos[0]));
```

Comment:
```suggestion
	const char* default_algos[] = {"md5", "sha1", "sha256", "crc32", "entropy"};
	bin->default_hashes = rz_list_new_from_array(default_algos, RZ_ARRAY_SIZE(default_algos));
```

Reply:
Thanks for the feedback, I have included your changes in my last commit. Let me know if anything else need changing.